### PR TITLE
HLint, use quasi-quoter

### PR DESCRIPTION
**Pull request by pbrisbin**
There are two HLint suggestions I ignored, both around reducing duplication. I
want think a bit more about how to tackle those, or if it's even necessary.

I did not test-compile locally; there is a Travis suite that scaffolds and
test-compiles right?

### CODE REVIEW

File: test/TestImport.hs [hunk 1]
```diff
@@ -36,16 +37,20 @@ withApp = before $ do
...
-    tables <- rawSql "SELECT table_name FROM information_schema.tables WHERE table_schema = 'public';" []
-    return $ map unSingle tables
\ No newline at end of file
+    tables <- rawSql [st|
+        SELECT table_name
+        FROM information_schema.tables
+        WHERE table_schema = 'public';
+    |] []
+
+    return $ map unSingle tables
```
Comment (MaxGabriel) [inline test/TestImport.hs]: Based on running `cabal test` on this branch, `shakespeare` needs to be added to the build-depends of the test project to import `Text.Shakespeare.Text`. 

Also, not huge deal, but it does make the SQL log output a little uglier:

```
14/Mar/2015:14:52:16 -0700 [Debug#SQL] "\n        SELECT table_name\n        FROM information_schema.tables\n        WHERE table_schema = 'public';\n    " [] @(persistent-2.1.1.4:Database.Persist.Sql.Raw ./Database/Persist/Sql/Raw.hs:42:26)
```

**Comment by MaxGabriel** [comment]
On the changes around `wipeDB`, each of the different database backends unfortunately has to use different code to wipe the databases. I think the ideas behind these changes would apply to the MySQL and possibly SQLite branches. (If you aren't already doing it, I'll make PRs for the other branches after your PR is merged).

**Comment by pbrisbin** [comment]
Added the test dependency. If you can handle the other backends, that'd be great.

File: test/Handler/CommonSpec.hs [hunk 2]
```diff
@@ -11,7 +11,7 @@ spec = withApp $ do
         it "has correct User-agent" $ do
             get RobotsR
             bodyContains "User-agent: *"
-    describe "favicon.ico" $ do
+    describe "favicon.ico" $
```
Comment (snoyberg) [inline test/Handler/CommonSpec.hs]: I'd probably ignore this hlint advice, as this still makes it harder to add tests in the future with minimal line-diffs.

**Comment by snoyberg** [comment]
:+1:

**Pull request merged**